web/components/application/hacker/pages/PageThree/PageThree.js

```diff
@@ -0,0 +1,135 @@
+              certain application/registration information for event administration, ranking,
+              MLH administration, and occasional messages about hackathons in line with
+              the <a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf" target="_blank" rel="noopener noreferrer">MLH Code of Conduct.</a>
+            </div>)}
+          className="margin-top-s dir-row margin-bottom-giga"
+        >
+          <Checkbox
+            label={(
+              <div>I authorize nwPlus to share certain application/registration information
+                for event administration, ranking, MLH administration, and occasional messages
+                about hackathons in line with
+                the <a href="https://mlh.io/privacy" target="_blank" rel="noopener noreferrer"> MLH Privacy Policy.</a>
+              </div>)}
+            value="isPrivacyPolicyChecked"
+            checked={isPrivacyPolicyChecked}
+            onChange={(e) => {
+              this.updateApplication({ isPrivacyPolicyChecked: e.target.checked });
+            }}
+          />
+          <Checkbox
+            label={(
+              <div className="bold">I have read and agree to
+                the <a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf" target="_blank" rel="noopener noreferrer"> Major League Hacking Code of Conduct.</a>
+              </div>
+            )}
+            value="isCodeOfConductChecked"
+            checked={isCodeOfConductChecked}
+            onChange={(e) => {
+              this.updateApplication({ isCodeOfConductChecked: e.target.checked });
+            }}
+          />
+        </CheckboxGroup>
+
+        <CheckboxGroup
+          name="data-checkbox"
+          label={(
+            <div><img className="vertical-align-top emoji" alt="🏅" src={floppy} /> We use
+              your (anonymized!) data to help you get the best sponsors and continuously improve
+              nwHacks with each iteration.
+            </div>
+          )}
+          className="margin-top-s dir-row margin-bottom-giga"
+        >
+          <Checkbox
+            label="I authorize nwPlus to use my anonymized data for data reporting."
+            value="isDataReportingChecked"
+            checked={isDataReportingChecked}
+            onChange={(e) => {
+              this.updateApplication({ isDataReportingChecked: e.target.checked });
+            }}
+          />
+        </CheckboxGroup>
+
+        <CheckboxGroup
+          name="documents-checkbox"
+          label={(
+            <div><img className="vertical-align-top emoji" alt="🏅" src={briefcase} /> Our hackathon
+              aims to connect you with industry professionals, recruiters, and career opportunities.
+              In doing so, information about our hackers is needed in order for attending companies
+              to contact you.
+            </div>)}
+          className="margin-top-s dir-row margin-bottom-giga"
```

Comment:
We can get rid of the `margin-bottom-giga` here to avoid too much whitespace before the submit button

Reply:
Okay, done